salt/bugs/config/postfix/main.cf

```diff
@@ -24,8 +24,8 @@ compatibility_level = 3.6
 
 
 # TLS parameters
-smtpd_tls_cert_file=ssl_certificate /etc/ssl/private/bugs.psf.io.pem;
-smtpd_tls_key_file=etc/ssl/private/bugs.psf.io.pem;
+smtpd_tls_cert_file=/etc/ssl/private/bugs.psf.io.pem
```

Comment:
This would now be `/etc/ssl/private/acme-bugs.python.org.pem` right?

Reply:
[0da94f1](https://github.com/python/psf-salt/pull/558/commits/0da94f14147c0b1f2556b4a7a7a7214e53ea6bbe)